linalg/operator.hpp

```diff
@@ -113,6 +113,18 @@ class Operator
        forms, though currently @a b is not used in the implementation. */
    virtual void RecoverFEMSolution(const Vector &X, const Vector &b, Vector &x);
 
+   /** @brief Return in @A a parallel (on truedofs) version of this operator.
+
+       This has some of the functionality of Operator::FormLinearSystem(), but
+       works additionally on rectangular matrices.
+
+       The constraints are specified through the prolongation P from
+       GetProlongation(), and restriction R from GetRestriction() methods, which
+       are e.g. available through the (parallel) finite element space of any
+       (parallel) bilinear form operator. We have: `A(X)=[R (*this)
+       P](X)`. */
+   void FormParallelOperator(Operator* &A);
+
```

Comment:
In the current `ParBilinearForm` this is called `FormSystemMatrix`:
```c++
/// Form the linear system matrix @a A, see FormLinearSystem() for details.
void FormSystemMatrix(const Array<int> &ess_tdof_list, OperatorHandle &A);
```

In the new `okina-integ` branch (see  #729) it is called `FormSystemOperator`:
```c++
/// Form the linear system operator @a A, see FormLinearSystem() for details.
virtual void FormSystemOperator(const Array<int> &ess_tdof_list,
                                Operator *&A);
```
I personally prefer the latter.